subprojects/gst-docs/examples/tutorials/README.md

```diff
@@ -0,0 +1,14 @@
+# Building Tutorials with Meson for VSCode
+
+Follow these steps to build the tutorials using Meson for VSCode:
```

Comment:
These steps aren't really related to VSCode or any other IDE.

Reply:
I thought I had also pushed the .vscode folder... but I now understand that this is not what should be done.